scripts/cloud-utils.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# set exit on error to true
+set -e
+# load .env
+set -o allexport
+source ./scripts/.env set
+
+REGION=us-east-1
+CURR_BRANCH=$(git branch --show-current)
+
+function authenticate {
+    account_number=$1
+    role_name=$2
+    profile_name=$3
+    echo Authenticating terminal...
+    mwinit
```

Comment:
When off VPN `--aea` flag is need to successfully authenticate. It doesn't appear to have ill effects to have this flag when on VPN, so best to add for convenience. 

```suggestion
    mwinit --aea
```